SerialPrograms/Source/PokemonSV/Programs/Sandwiches/PokemonSV_SandwichRoutines.cpp

```diff
@@ -637,8 +637,9 @@ void make_two_herbs_sandwich(
 
 void make_sandwich_option(ProgramEnvironment& env, ConsoleHandle& console, BotBaseContext& context, SandwichMakerOption& SANDWICH_OPTIONS){
     const Language language = SANDWICH_OPTIONS.LANGUAGE;
-    if (language == Language::None){
-        throw UserSetupError(console.logger(), "Must set game langauge option to read ingredient lists.");
+
+    if (language == Language::None) {
+        throw UserSetupError(console.logger(), "Must set game language option to read ingredient lists.");

```

Comment:
This might've crossed with a global code formatting change yesterday where I removed spaces between ")" and "{"